rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1295,15 +1302,8 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 		const auto mp = lv2_fs_object::get_mp("/dev_hdd0");
 
-		fs::device_stat info;
-		if (!fs::statfs(vfs::get("/dev_hdd0"), info))
-		{
-			sys_fs.error("sys_fs_fcntl(0xc0000002): unexpected error %s", fs::g_tls_error);
-			return CELL_EIO; // ???
-		}
-
 		arg->out_block_size = mp->block_size;
-		arg->out_block_count = info.avail_free / mp->block_size;
+		arg->out_block_count = (40ull * 1024 * 1024 * 1024 - 1) / mp->block_size; // Read explanation in cellHddGameCheck
```

Comment:
at least fix it in cellFsGetFreeSize as well

Reply:
unless avail_free is in bytes anyway. whatever

Reply:
I will just rewrite cellFsGetFreeSize with sys_fs implementation to avoid this.

Reply:
Done, and no its in blocks.